go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/containerd/stargz-snapshotter/estargz v0.4.1 // indirect
 	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
 	github.com/docker/cli v20.10.5+incompatible // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/distribution v2.8.0+incompatible // indirect
```

Comment:
Why didn't we update to the next patch version? 2.8.1?

Reply:
no particular reason, I usually take the next version I know exists. But if there is a 2.8.1 I guess it will be fine too.

Reply:
bumped to v2.8.1